content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -206,6 +206,8 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
   const [stopNumber, setStopNumber] = useState(stopNumberServerSide);
   const [currentStop, setCurrentStop] = useState(currentStopServerSide);
   const [headerEl, setHeaderEl] = useState<HTMLElement>();
+  const guideTypeUrl = `/guides/exhibitions/${exhibitionGuideId}/${type}`;
```

Comment:
Not for this PR but for https://github.com/wellcomecollection/wellcomecollection.org/pull/11190, this will need to be considered in there 🙌  (@gestchild )